easybuild/easyconfigs/f/FreeTDS/FreeTDS-1.3.3-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,41 @@
+# easybuild easyconfig
+#
+easyblock = 'ConfigureMake'
+
+name = 'FreeTDS'
+version = '1.3.3'
+
+homepage = "https://www.freetds.org/"
+description = """FreeTDS is a set of libraries for Unix and Linux that allows your programs to
+ natively talk to Microsoft SQL Server and Sybase databases."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://www.freetds.org/files/stable']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a21a0aa351185049e11e237a795a61e3ca68d2e798259b3b5ea4a9797d5a5535']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('libreadline', '8.1'),
+]
+
+# osdependencies = [OS_PKG_OPENSSL_DEV]
```

Comment:
@scicomp-moffitt Any reason this is commented out? Is OpenSSL required? If so, it should be added as a proper dependency now through the `OpenSSL` wrapper we have:

```python
dependencies = [
    ('libreadline', '8.1'),
    ('OpenSSL', '1.1', '', True),
]

Reply:
@scicomp-moffitt any updates here?